instances/surveys/docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: '3'
+
+networks:
+  web:
+    external: true
+  database:
+    external: true
```

Comment:
Remove this as we don't need this container to know about our other database (since this limesurvey has its own mysql running within itself).